### PR TITLE
Cluster tools annotation

### PR DESCRIPTION
**Pull request by brendarearden**
**Issue**
rancher/rancher#33841

**Problem**
new Cluster Tools page needs a way to display specific charts.

**Solution**
Pull in new backup restore operator tag that has the `catalog.cattle.io/type=cluster-tool` chart annotation

**Related PRs**
https://github.com/rancher/charts/pull/1388
https://github.com/rancher/backup-restore-operator/pull/139

### CODE REVIEW

**Comment by jiaqiluo** [comment]
@brendarearden  can you remove the rc6 chart from `charts/` `assets/` and `index.yaml` ?

**Review by jiaqiluo — APPROVED** [review]
@brendarearden  can you remove the rc6 chart from `charts/` `assets/` and `index.yaml` ?

**Review by jiaqiluo — CHANGES_REQUESTED** [review]
Oops, I hit the wrong button. Please check my previous comment :-)

File: index.yaml [hunk 1]
```diff
@@ -936,23 +939,23 @@ entries:
...
     description: Installs the CRDs for rancher-backup.
-    digest: 0c209bcf63c6b19a85fa9eaba1ef1ed1aa3271ef1c68d08f5e452d78fc38edb1
+    digest: 9bb076badc71ec5fba6135f192eda6c01a269b85e0971e4fe2921cdc76865d93
     name: rancher-backup-crd
     type: application
     urls:
-    - assets/rancher-backup-crd/rancher-backup-crd-2.0.0+up2.0.0-rc6.tgz
-    version: 2.0.0+up2.0.0-rc6
+    - assets/rancher-backup-crd/rancher-backup-crd-2.0.0+up2.0.0-rc7.tgz
+    version: 2.0.0+up2.0.0-rc7
```
Comment (jiaqiluo) [inline index.yaml:950]: this is the change we are looking for.

File: index.yaml [hunk 2]
```diff
@@ -795,21 +797,22 @@ entries:
...
     icon: https://charts.rancher.io/assets/logos/backup-restore.svg
     keywords:
     - applications
     - infrastructure
     name: rancher-backup
     urls:
-    - assets/rancher-backup/rancher-backup-2.0.0+up2.0.0-rc6.tgz
-    version: 2.0.0+up2.0.0-rc6
+    - assets/rancher-backup/rancher-backup-2.0.0+up2.0.0-rc7.tgz
+    version: 2.0.0+up2.0.0-rc7
```
Comment (jiaqiluo) [inline index.yaml:815]: this is the change we are looking for.

**Review by jiaqiluo — DISMISSED** [review]

File: index.yaml [hunk 3]
```diff
@@ -371,12 +371,13 @@ entries:
       catalog.cattle.io/os: linux
       catalog.cattle.io/provides-gvr: longhorn.io/v1beta1
       catalog.cattle.io/release-name: longhorn
+      catalog.cattle.io/type: cluster-tool
```
Comment (aiyengar2) [inline index.yaml:374]: annotation was bumped as expected

**Review by aiyengar2 — APPROVED** [review]
LGTM

File: index.yaml (hunk 3 above)
Comment (cbron) [reply index.yaml:374]: Why was it bumped now and not in previous PR that actually did the change ?

Comment (aiyengar2) [reply index.yaml:374]: @cbron , @brendarearden noted in https://github.com/rancher/charts/pull/1388 that she would be doing the cluster tool bump for backup in a separate PR.

Comment (cbron) [reply index.yaml:374]: This one is for longhorn, there are ones here for monitoring too. Did we forget to regenerate the index in the other PR ?

**Review by cbron — DISMISSED** [review]
Approving because this should get us to the correct state, but can we figure out why the index didn't change in first PR.

File: index.yaml (hunk 3 above)
Comment (brendarearden) [reply index.yaml:374]: Usually `make charts` updates the index.yaml, I also have been regenerating the index.yaml, but maybe during a rebase I removed it. I went ahead and regenerated and push it up in its own commit.

**Review by cbron — APPROVED** [review]

**Pull request merged**